components/skus/browser/rs/lib/src/models.rs

```diff
@@ -186,6 +190,21 @@ pub struct SingleUseCredentials {
     pub issuer_id: Option<String>,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct TimeLimitedV2Credential {
+    pub unblinded_creds: Option<Vec<SingleUseCredential>>,
+    pub issuer_id: Option<String>,
+    pub valid_from: NaiveDateTime,
+    pub valid_to: NaiveDateTime,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct TimeLimitedV2Credentials {
+    pub item_id: String,
+    pub tokens: Vec<Token>,
```

Comment:
can we please change this terminology to align with the single use creds above?
`s/creds/unblinded_creds`
`s/tokens/creds`